c2corg_ui/static/js/utils.js

```diff
@@ -121,21 +121,18 @@ app.utils.getImageFileBase64Source = function(file) {
 app.utils.createImageSlide = function(file, imageUrl) {
   var img;
   var ahref;
-  // if !src => it's an already existing image, else it's an image that has just been uploaded
-  if (!file['src']) {
-    ahref = '<a href="' + imageUrl + file.filename + '" data-info-id="' + file['document_id'] + '">';
-    img = '<img src="' + imageUrl + file.filename + '">';
-  } else {
-    ahref = '<a href="' + file['src'] + '" data-info-id="' + file['document_id'] + '">';
-    img = '<img src=" ' + file['src'] + '">';
-  }
-  return '<figure id="image-' + file['document_id'] + '">' +
-               ahref + img +
-               '</a>' +
-               '<app-slide-info></app-slide-info>' +
-             '</figure>';
-};
+  var smallImage = app.utils.createImageUrl(file.filename, 'SI');
+  var bigImage = app.utils.createImageUrl(file.filename, 'BI');
 
+  ahref = '<a href="' + imageUrl + bigImage + '" data-info-id="' + file['image_id'] + '-slide">';
+  img = '<img src="' + imageUrl + smallImage + '">';
+
+  return '<figure id="' + file['image_id'] + '">' +
+              ahref + img +
+              '</a>' +
```

Comment:
That would make more sense to directly embed the ``<a>`` and ``<img>`` code in the HTML above, that would be more readable. And also more consistent. For instance ``ahref`` contains the opening ``<a>`` tag whereas the closing tag is in the HTML above.